### PR TITLE
New package r-rapiserialize at 0.1.0

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-rapiserialize/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    This package provides other packages with access to the internal R
+    serialization code. Access to this code is provided at the C function
+    level by using the registration of native function mechanism. Client
+    packages simply include a single header file RApiSerializeAPI.h provided
+    by this package. This packages builds on the Rhpc package by Junji Nakano
+    and Ei-ji Nakama which also includes a (partial) copy of the file
+    src/main/serialize.c from R itself. The R Core group is the original
+    author of the serialization code made available by this package."""
+
+    homepage = "https://cran.r-project.org/web/packages/RApiSerialize/index.html"
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-rapiserialize/package.py]:
```suggestion
    homepage = "https://cloud.r-project.org/web/packages/RApiSerialize/index.html"
```

File: var/spack/repos/builtin/packages/r-rapiserialize/package.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    url      = "https://cloud.r-project.org/src/contrib/RApiSerialize_0.1.0.tar.gz"
+
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-rapiserialize/package.py:24]: Need `list_url`.

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-rapiserialize/package.py (hunk 2 above)
Comment (dorton21) [reply var/spack/repos/builtin/packages/r-rapiserialize/package.py:24]: No archive link is listed for this package.

Comment (glennpj) [reply var/spack/repos/builtin/packages/r-rapiserialize/package.py:24]: That will get created once the next release of the package is out. Eventually, the `list_url` will be automated, #21861, and will not need to be entered. For now, use this:
```
list_url = "https://cloud.r-project.org/src/contrib/Archive/RApiSerialize"
```

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-rapiserialize/package.py (hunk 2 above)
Comment (dorton21) [reply var/spack/repos/builtin/packages/r-rapiserialize/package.py:24]: Thanks

**Review by glennpj — APPROVED** [review]

**Pull request merged**